drag_and_drop_v2/templates/html/drag_and_drop_edit.html

```diff
@@ -31,6 +31,9 @@
                     </select>
                     <span class="sr">{{ help_texts.mode }}</span>
 
+                    <label class="h3 setting assessment" for="max-attempts" title="{{ help_texts.max_attempts }}">{% trans "Maximum attempts" %}</label>
```

Comment:
If you nest the input within the label, there is no need for the "for" attribute and the id.